### DISCUSSION TITLE
How to roll back from v0.10.5 (yanked) to v0.10.4?

### DISCUSSION THREAD

**Discussion by Yorden** [eqlabs/pathfinder#1741] [Q&A] [answered]
Updated the node with the image to v0.10.5

There is an issue with it. I got to go back to v0.10.4

How do I proceed ?

Thanks

**Comment by Mirko-von-Leipzig** [comment]
Unfortunately v0.10.5 is not trivially possible to roll-back. This is part of the reason we yanked the release - its quite broken I'm afraid. We sincerely apologise for this.

A couple of questions just to check if its possible fix.

1. Did you actually run the v0.10.5 node?
  1. If yes, did it complete the database migration step?

**Reply by tomynocker** [reply]
Can you go back if a database has migrated to version 47 and gives this error when updating to version v.0.10.6?:
"Database version 47 is newer than this application expected 45"

**Reply by Mirko-von-Leipzig** [reply]
No unfortunately not. 

However, what you can try is use our `v0.11.0-rc0` pre-release which should work.

**Reply by tomynocker** [reply]
Thanks!

**Comment by Mirko-von-Leipzig** [comment, marked as answer]
You can now use the `v0.11.0-rc0` pre-release which should be okay and working.
